spec/acceptance/nodesets/default.yml

```diff
@@ -6,9 +6,9 @@ HOSTS:
     pe_upgrade_dir:
     pe_upgrade_ver:
     hypervisor: vmpooler
-    platform: windows-2012-64
+    platform: windows-2012r2-64
     ruby_arch: x64
-    template: win-2012-x86_64
+    template: win-2012r2-wmf5-x86_64
```

Comment:
I changed my default to run wmf5 because dsc needs powershell 5... not sure what the defaults should be for chocolatey